lib/ansible/modules/cloud/ovirt/ovirt_vms.py

```diff
@@ -606,7 +606,7 @@ def __get_template_with_version(self):
                         self.param('template_version')
                     )
                 )
-            template = templates[0]
+            template = sorted(templates,key=lambda t: t.version.version_number)[-1]
```

Comment:
Can you please add space after ',' and also use `reverse=True` with `[0]` rather then [-1], I think it's more clear.

Reply:
Sure, i'll do